docs/1-introduction/1-1-getting-started/1-1-3-hello-triangle.md

```diff
@@ -102,19 +135,35 @@ PSOutput Main(PSInput input)
 
 Here as well we have an input `PSInput`, and an output `PSOutput`.
 
-Since there aren't any other shaders in between the VS and the PS, the VS's output is the PS's input, the naming might be a bit confusing but that's the gist of it, PSInput should match the VSOutput in vertex shader, this isn't entirely required but not doing so is only advisable if you really know what you are doing.
+Since there aren't any other shaders in between the VS and the PS, the VS's 
```

Comment:
> Since there aren't any other shaders in between the VS and the PS

Perhaps reword this to: "Since we haven't configured any other shader stages in between the VS and PS ..."
as this implies there are no other stages, even though there are quite a few left :)

Reply:
100% agree - this is a miscommunication